pkg/commands/alerts/create.go

```diff
@@ -0,0 +1,121 @@
+package alerts
+
+import (
+	"io"
+
+	"github.com/fastly/go-fastly/v9/fastly"
+
+	"github.com/fastly/cli/pkg/argparser"
+	fsterr "github.com/fastly/cli/pkg/errors"
+	"github.com/fastly/cli/pkg/global"
+)
+
+// NewCreateCommand returns a usable command registered under the parent.
+func NewCreateCommand(parent argparser.Registerer, g *global.Data) *CreateCommand {
+	c := CreateCommand{
+		Base: argparser.Base{
+			Globals: g,
+		},
+	}
+	c.CmdClause = parent.Command("create", "Create Alerts")
+	c.CmdClause.Flag("name", "Name of the alert definition").Required().StringVar(&c.name)
+	c.CmdClause.Flag("description", "Description of the alert definition").Required().StringVar(&c.description)
+	c.CmdClause.Flag("metric", "Metric to alert on").Required().StringVar(&c.metric)
+	c.CmdClause.Flag("source", "Source to get the metric from").Required().StringVar(&c.source)
+	c.CmdClause.Flag("type", "Type of evaluation strategy, one of [all_above_threshold, above_threshold, below_threshold, percent_absolute, percent_decrease, percent_increase]").Required().StringVar(&c.eType)
+	c.CmdClause.Flag("period", "Period of time to evaluate, one of [2m, 3m, 5m, 15m, 30m]").Required().StringVar(&c.period)
```

Comment:
Wherever explicit/predefined inputs are expected, use an `EnumVar` along with`.HintOptions`, see the following for an example:

https://github.com/fastly/cli/blob/main/pkg/commands/serviceauth/create.go#L40

Reply:
Done